### DISCUSSION TITLE
[Bug]文件夹模式显示歌曲数量不足

### DISCUSSION THREAD

**Discussion by junilask** [gitbobobo/StreamMusic#455] [Q&A] [answered]
使用直连模式连接Navidrome,文件夹模式展示的文件夹数量和歌曲数量都只有个位数

**Comment by gitbobobo** [comment, marked as answer]
navi没有文件夹接口，所以展示的是本地数据。

或许我应该在直连模式下隐藏不支持文件夹接口的服务端的文件夹入口？

**Reply by junilask** [reply]
或许，毕竟相当于在直连模式下此功能不能支持呢....
现在绝大多数都是使用的Navi方案，请问其他音乐服务器方案中有支持文件夹接口的吗？

**Reply by gitbobobo** [reply]
官网有音乐服务的功能对比
